argv.js

```diff
@@ -1,9 +1,23 @@
 const fs = require("fs");
 const path = require("path");
 const commandLineArgs = require("command-line-args");
+const pluralize = require("pluralize");
```

Comment:
I'd rather not add trivial dependencies. A simple "Unknown arguments: ${flags}" will do.

Reply:
Cool, will throw it out